internal/backends/sqlite/metadata/registry.go

```diff
@@ -352,6 +354,27 @@ func (r *Registry) CollectionRename(ctx context.Context, dbName, oldCollectionNa
 	panic("not implemented")
 }
 
+// Version returns SQLite version.
+//
+// If there are no databases in the registry, empty string is returned.
+func (r *Registry) Version(ctx context.Context) (string, error) {
+	if r.version != "" {
+		return r.version, nil
+	}
+
+	db := r.p.GetFirst(ctx)
+	if db == nil {
+		return "", nil
+	}
+
+	row := db.QueryRowContext(context.Background(), "SELECT sqlite_version()")
+	if err := row.Scan(&r.version); err != nil {
```

Comment:
Do we need a lock for writing to the registry? :thinking:

Reply:
Good question! Version is a minor thing compared to databases, so I don't want to lock the whole registry with the existing mutex. A new mutex that only protects this field can be set, but again I'm not sure if it brings a lot of value. In principle, I can set it to be completely accurate.

Reply:
Okay, let's leave it as it is for now. :+1: